features/nvstore/README.md

```diff
@@ -0,0 +1,58 @@
+# NVStore
```

Comment:
This looks like a great start for the docs! Is there a copy in the handbook or is this best placed there?

Reply:
Thanks. Regarding your question about documentation in the handbook - yes, there is such copy. It is currently handled in a separate PR. There's also a new example repo (https://github.com/ARMmbed/mbed-os-example-nvstore). Note that both can't be finalized until this PR is merged in master.

Reply:
Can you link or reference the docs PR to this one? Would like to make sure they go in together.

Reply:
[Here it is](https://github.com/ARMmbed/Handbook/pull/399).

Reply:
depends on https://github.com/ARMmbed/Handbook/pull/399

Reply:
Isn't making this PR depend on the documentation PR problematic? As far as I managed to understand from @AnotherButler , I can only complete the Handbook PR after publishing my example code via the online compiler. I have an example repo, which points to this branch, but will only be able to build it in the online compiler after the NVStore gets into the master branch (as the online compiler doesn't see the NVStore code). So we have the chicken and egg situation here. What am I missing?

Reply:
I'll edit the comment. I was trying to get github to make a nice link graphic but it didn't work.